mailinglogger/datatypes.py

```diff
@@ -23,25 +23,27 @@ def headers(self):
             return self.section.headers.values
         else:
             return {}
-        
+
     def create_loghandler(self):
+        secure = () if self.section.use_tls else None
         return MailingLogger(self.section.fromaddr,
                              self.section.toaddrs,
-                             self.mailhost(),
-                             self.section.subject,
-                             self.section.send_empty_entries,
-                             self.section.flood_level,
-                             self.section.username,
-                             self.section.password,
-                             self.section.ignore,
-                             self.headers(),
-                             self.section.template,
-                             self.section.charset,
-                             self.section.content_type)
+                             mailhost=self.mailhost(),
```

Comment:
It's about consistency with MailingLoggerHandlerFactory.create_loghandler method, a for code auto-documentation